### PR TITLE
Removed Personalisation

### PR DESCRIPTION
**Pull request by sparrow8332**
Removed Personalisation and Renamed mod appropriately

### CODE REVIEW

File: CQUI.modinfo [hunk 1]
```diff
@@ -127,6 +62,7 @@
     <File>Assets/Text/cqui_text_settings_ru.xml</File>
     <File>Assets/Text/cqui_text_settings_zh.xml</File>
     <File>Assets/Text/cqui_text_settings_es.xml</File>
+	<File>Assets/Text/Gossip_Text.xml</File>
```
Comment (Azurency) [inline CQUI.modinfo]: Little detail : the spacing is wrong here

File: CQUI.modinfo [hunk 2]
```diff
@@ -170,7 +106,6 @@
     <File>Assets/UI/Panels/unitpanel.xml</File>
     <File>Assets/UI/PartialScreens/citystates.lua</File>
     <File>Assets/UI/PartialScreens/citystates.xml</File>
-    <File>Assets/UI/Popups/boostunlockedpopup.lua</File>
```
Comment (Azurency) [inline CQUI.modinfo]: why ? I added this this

File: CQUI.modinfo [hunk 3]
```diff
@@ -299,7 +233,6 @@
         <File>Assets/UI/Panels/unitpanel.xml</File>
         <File>Assets/UI/PartialScreens/citystates.lua</File>
         <File>Assets/UI/PartialScreens/citystates.xml</File>
-        <File>Assets/UI/Popups/boostunlockedpopup.lua</File>
```
Comment (Azurency) [inline CQUI.modinfo]: same here, I added this in 90bc06981fe68f8c9724e107ec5d7971786d866f

**Comment by sparrow8332** [comment]
#072b526   Do i have to make a new pull request ?

**Comment by sparrow8332** [comment]
Changes made in 89ed038

File: CQUI.modinfo (hunk 3 above)
Comment (sparrow8332) [reply CQUI.modinfo]: changes made

**Pull request merged**